src/content/docs/kubernetes-pixie/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx

```diff
@@ -28,6 +28,7 @@ In addition:
 
 * Review this [Pixie data security overview](/docs/auto-telemetry-pixie/pixie-data-security-overview) for actions to take to secure your data.
 * Make sure you have sufficient memory: Pixie requires, at least, 2Gb of memory on each node in your cluster. More memory may be required for larger clusters.
+* Pixie requires elevated privileges. By default, installing the NR Kubernetes integration with privileged=falese will also prevent Pixie from running. See [Step 3 here](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#installing-and-configuring-nri-bundle-with-helm) 
```

Comment:
Looks good.  Please update spelling `privileged=falese` to `privileged=false`. :)

Reply:
I'll fix with some minor styleguide edits